tsconfig.json

```diff
@@ -1,33 +1,22 @@
 {
-  // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
   "include": ["src", "types"],
   "compilerOptions": {
     "module": "esnext",
     "lib": ["dom", "esnext"],
     "importHelpers": true,
-    // output .d.ts declaration files for consumers
     "declaration": true,
-    // output .js.map sourcemap files for consumers
     "sourceMap": true,
-    // match output dir to input dir. e.g. dist/index instead of dist/src/index
-    "rootDir": "./src",
-    // stricter type-checking for stronger correctness. Recommended by TS
+    // "rootDir": "./src",
```

Comment:
(from txstate-port) I think removing `"rootDir": "./src"` will affect the build (?), but I want to type-check "types" folder too. How to go about this?